src/sentry/explore/models.py

```diff
@@ -60,10 +60,13 @@ class ExploreSavedQuery(DefaultFieldsModel):
         choices=ExploreSavedQueryDataset.as_choices(), default=ExploreSavedQueryDataset.SPANS
     )
     is_multi_query = models.BooleanField(default=False)
+    prebuilt_id = BoundedPositiveIntegerField(null=True, db_default=None)
```

Comment:
We should probably leave a comment here defining what this is for because it's not immediately clear how prebuilts are managed/included in this model